src/components/Toast/useToast.tsx

```diff
@@ -7,7 +7,7 @@ export interface UseToastProps {
 export function useToast(): UseToastProps {
   const { setNotice } = useToastContext();
   const showToast = useCallback(
-    (props) => {
+    (props: any) => {
```

Comment:
**suggestion:** Could be `ToastNoticeProps`
```suggestion
    (props: ToastNoticeProps) => {
```

Reply:
:+1: